### ISSUE TITLE
Missing Datasets?

### ISSUE THREAD

**Issue by smk508** [waldronlab/curatedMetagenomicData#173]
Hi,

I noticed that some of the datasets which are listed on the curation page and have tsv files with metadata such as HMP_2019_t2d, HMP_2019_ibdmdb, and HallAB2017 are not accessible as far as I can tell.

If I run `ls("package:curatedMetagenomicData")` I don't see these packages listed. I also don't see them listed under `help(package="curatedMetagenomicData")`, and if I try to load these datasets (eg. `curatedMetagenomicData("HMP_2019_t2d.metaphlan_bugs_list.stool", dryrun=FALSE)`), I get an error.

Is there some version inconsistency or some command that I am unaware of to access these datasets?

I'm on R version 4.0.4 and bioconductor 3.12

Thanks,
Saad

**Comment by edoardopasolli** [comment]
Dear Saad, in the curation page we have a more updated list of datasets. They will be added to the package as soon as we will build and release the new version

**Comment by smk508** [comment]
> Dear Saad, in the curation page we have a more updated list of datasets. They will be added to the package as soon as we will build and release the new version

I see. Do you have an estimate on when the next version is planned for release?

Thanks,
Saad

**Comment by lwaldron** [comment]
Yes, I expect the major update to Biobakery 3 and these new datasets to be included in the upcoming release of Bioconductor 3.13. Processing is
